### ISSUE TITLE
move Boss casting particles

### ISSUE THREAD

**Issue by pausp1** [HiddenLair/Run4YourLife#633]
Hi!
I've noticed that the casting particles appears on the wrong hand (right) they should appear in the left hand, could you change this please?
Thank you!

**Comment by Moaif** [comment]
Nope, they appear on the correct one, the one that is nearest to the camera, and the one that our testplay revealed was more intuitive for this purpose.

**Comment by pausp1** [comment]
Put it in booth hands then, this particles was designed to be in the left hand, because is the hand that casts the skills. Maybe put it in the right hand has sense in gameplay but not in the artistic point of view so please make appear the particles in the left hand or in booth hands, but not only in the right one
